packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -242,6 +259,16 @@ export default async args => {
             // configs affected by a change => we cannot know in advance which one will be "the last".
             // So we run the same logic for all configs, letting the last one win.
             command([
+                async () => {
```

Comment:
Shall we extract duplicate logic into a module imported by both rollup configs? This part (and others) appears both in `rollup.config.js` and `rollup.config.native.js`.

Reply:
done